kubectl-volsync/cmd/migration_test_helper.go

```diff
@@ -0,0 +1,106 @@
+package cmd
```

Comment:
Code only used for/in tests needs to go in files named `*_test.go`

Reply:
sure